### PR TITLE
Add contributors.txt file

### PR DESCRIPTION
**Pull request by Acans**
As per the discussion in SMF Development, Arantor has suggested the use of a contributors text file to replace the credits. This file list everybody (I hope) that's either submitted code to SMF or been on the SMF Team. Also includes a small special thanks section and credits to third party software and graphics.

P.S: This is just a base for others to work off, most noticeable missing is SCEditor.

Signed-off-by: Acans acans@simplemachines.org

### CODE REVIEW

File: contributors.txt [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+Code Contributors; 
+
+Aaron van Geffen, AngelinaBelle, Antechinus, Antes, Bjoern "Bloc" Kristiansen, Brad "IchBin™" Grow, 
+Bryan "Runic" Deakin, Colin Schoen, Dr. Deejay, ElPlayer, emanuele, Feline, Graeme Spence, grafitus, gnif, 
+Hendrik Jan "Compuart" Visser, hhy89, Jessica González, Joshua "groundup" Dickerson, Jeremy "SleePy" Darwood, 
+Juan "JayBachatero" Hernandez, Karl "RegularExpression" Benson, Kays, live627, Marcus "cσσкιє мσηѕтєя" Forsberg, 
+Matt "Grudge" WolfMatthew "Labradoodle-360" Kerle, Michael "Oldiesmann" Eshom, Michele "Illori" Davis, NIBOGO, Norv,
```
Comment (eurich) [inline contributors.txt]: There's a missing comma > Matt "Grudge" WolfMatthew

Comment (Acans) [inline contributors.txt]: Indeed, corrected.

File: contributors.txt [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Software
+JQuery | © John Resig | Licensed under The MIT License (MIT)
+hoverIntent | © Brian Cherne | Licensed under The MIT License (MIT)
+Superfish | © Joel Birch | Licensed under The MIT License (MIT)
+SCEditor | © Sam Clarke | Licensed under The MIT License (MIT)
+animaDrag | © Abel Mohler | Licensed under The MIT License (MIT)
+
+Fonts
+Anonymous Pro |© 2009 | This font is licensed under the SIL Open Font License, Version 1.1
```
Comment (live627) [inline contributors.txt:67]: Why is there no space before this copyright symbol when there is in the paragraph above?

**Comment by live627** [comment]
Shouldn't this file be renamed to CONTRIBUTERS to keep in line with the standard package distributions/naming scheme?

**Comment by Arantor** [comment]
I'm not fussed whether it's called CONTRIBUTORS or CONTRIBUTES or whatever (noting full well we don't have our licence file called LICENSE or COPYING, and we don't have our readme as README, or even our changelog following any of the normal rules)

I'm just glad that we're getting there.

**Comment by ChainyEO** [comment]
"Contributes" is not possible, as it's a verb. It would have to be "Contributors" or "Contributions".

**Comment by Arantor** [comment]
You'd think so, but one of the standard files in 'standard naming schemes' is COPYING, also a verb, to reference what your rights are for copying. Yeah, there are some 'great' practices out there.

**Comment by live627** [comment]
It can't be CONTRIBUTES because it's not about the code submissions. Likewise, CONTRIBUTING won't work since it isn't about how to get involved. CONTRIBUTERS works best because it's a list of all who sent in code or support.

(My previous comment had a typo, sorry.)

**Comment by ChainyEO** [comment]
Seems that the generally accepted spelling of that word is contributORs, both in British and American English (presumably in other forms, too). I've just searched in Collins and Merriam-Webster (British and American versions). Both show the form with -OR.

**Comment by Arantor** [comment]
@ChainyEO I would not disagree with you. I'm just saying that those open source folks are kinda crazy :P And yes, typically they use the verb form rather than a more correct form (e.g. COPYING), though typically they use AUTHORS for such a thing.

But you know what? This file isn't that, this file says contributors.txt, it follows the consistency of other files we bundle, so I'm just going to merge it in and we can fix things like spacing afterwards.

**Pull request merged**